### PR TITLE
feat: added navigation component

### PR DESCRIPTION
**Pull request by srv-pranjal**
# Added Navigation Component

# Description

Navigation
 - Simple Navigation
 - Navigation with Search
 
 
image

### CODE REVIEW

File: pages/navigation.html [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+        </p>
+        <h2 class="component__sub-heading">Simple Navigation</h2>
+        <p class="component__sub-description">
+          A simple navigation bar can be build by using a <em>nav</em> with
+          class name <span class="highlight">nav</span> wrapped inside a
+          container with class name -
+          <span class="highlight">nav-wrapper</span>. <br />
+          Check out the below code to get started with a simple navigation bar.
+        </p>
+        <section class="component__display center-cross-axis">
```
Comment (akash-1318) [inline pages/navigation.html:94]: Use only one type for naming classes .

**Review by akash-1318 — COMMENTED** [review]
Just one suggestion added else everything looks good to me.

File: pages/navigation.html (hunk 1 above)
Comment (arjunbharti) [reply pages/navigation.html:94]: Follow same type of naming convention through out the code

**Review by arjunbharti — COMMENTED** [review]
Just one suggestion - Follow the same naming convention style. Everything else is fine!

File: pages/navigation.html (hunk 1 above)
Comment (srv-pranjal) [reply pages/navigation.html:94]: Fixed, thanks

**Pull request merged**